plugins/inputs/socket_listener/socket_listener.go

```diff
@@ -23,6 +23,8 @@ import (
 //go:embed sample.conf
 var sampleConfig string
 
+const defaultMaxDecodedSize = 500 * 1024 * 1024 //500MB
```

Comment:
Should we declare this in a more central place e.g. in `internal/content_coding.go`?